TechliftTelegramBot/Services/PlayerInfoService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+
+using System.Globalization;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using TechliftTelegramBot.Models;
+using TechliftTelegramBot.Services;
+
+namespace TechliftTelegramBot.Services
+{
+    
+    public class PlayerInfoService:IPlayerInfoService
+    {
+        private readonly HttpClient _client;
+        private List<Player> player = new();
+        private readonly ApplicationConfiguration _config;
+        public PlayerInfoService(IOptions<ApplicationConfiguration> config, IHttpClientFactory client)
+        {
+            _config = config.Value;
+            _client = client.CreateClient();
```

Comment:
again dont create and keep the httpclient, httpclient should always be used per request, store the factory and create client whenever needed